charts/s1-agent/templates/agent/daemonset.yaml

```diff
@@ -36,7 +36,11 @@ spec:
         image: "{{ include "agent.full_url" . }}"
         imagePullPolicy: {{ default "IfNotPresent" .Values.configuration.imagePullPolicy }}
         resources:
+{{- if .Values.configuration.env.injection.enabled }}
+{{ toYaml .Values.agentInjection.resources | indent 10 }}
+{{- else }}
```

Comment:
what will happen in hybrid cluster?

Reply:
@arields1
 that is an excellent point!
The daemonset is used only on non-fargate nodes, so it should be the resources specified in the agent section.
the injected agent takes its config from the configmap, so there you should take the injected section